src/lib/core/components/visualizations/implementations/ScatterplotVisualization.tsx

```diff
@@ -363,6 +363,7 @@ function ScatterplotViz(props: Props) {
               labelPlacement={'end'}
               // minWidth is used to set equivalent space per item
               minWidth={210}
+              // minWidth={75}
```

Comment:
I think this comment can be removed

Reply:
@dmfalke yep. I tested the spacing but forgot to remove it - removed